test_unstructured_ingest/metrics/all-docs-cct.tsv

```diff
@@ -1,4 +1,4 @@
 filename	connector	cct-accuracy	cct-%missing
-science-exploration-1p.pptx	box	0.861	0.09

```

Comment:
so is the science-exploration file being removed now?

Reply:
was just wondering the reason for the change on this file

Reply:
This file was probably existed on my local but not during the test-ingest-xx. So when I pulled and pushed again it got removed.

Reply:
ok, so is the diff change from `science-exploration-1p.ppt` to `handbook-1p.docx` expected? just want to make sure we have the right/expected comparison file in place so I'm deferring to you

Reply:
Yes, considering some of the file existed locally and got pushed to the branch, while some of the files are available running from CI only.